src/components/views/read/cvForm/Review.tsx

```diff
@@ -0,0 +1,272 @@
+import { FC, useState } from 'react';
+
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import DownloadIcon from '@mui/icons-material/Download';
+import NavigateBeforeIcon from '@mui/icons-material/NavigateBefore';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Grid,
+  Typography,
+} from '@mui/material';
+
+interface InnerObject {
+  [key: string]: string;
+}
+
+interface ValuesObject {
+  [key: string]: InnerObject;
+}
+interface CVValues {
+  [category: string]: {
+    [fieldName: string]: any;
+  }[];
+}
+interface Props {
+  nextPage: () => void;
+  prevPage: () => void;
+  homeStep: () => void;
+  prevStep: () => void;
+  values: ValuesObject;
+}
+const Review: FC<Props> = ({
+  nextPage,
+  prevPage,
+  homeStep,
+  prevStep,
+  values,
+}) => {
+  const handleNext = (): void => {
+    nextPage();
+    homeStep();
+  };
+  const handlePrev = (): void => {
+    prevPage();
+    prevStep();
+  };
+  const [cards, setCards] = useState([{ id: 1 }]);
+  const handleDownload = (): void => {
+    const CvGenerated = 'link';
+    <a
+      href={CvGenerated}
+      download="Example-PDF-document"
+      target="_blank"
+      rel="noreferrer"
+    >
+      <Button type="button">Download .pdf file</Button>
+    </a>;
+  };
+  const personalData = values['Personal Info'];
+  const educationData = values.Education;
+  const workData = values['Work Experience'];
+  const skillsData = values.Skills;
+  const portfolioData = values.Portfolio;
+  const motivationData = values['Motivation Letter'];
+  const referencesData = values.References;
```

Comment:
This style is not coherent, it would be better to use "machine names" like "education", "personalInfo", "motivationLetter" as object keys and then be able to map them to a human label, either using a mapping that you define of with translations.